pages/api/team-member/[mail].tsx

```diff
@@ -0,0 +1,12 @@
+import prisma from '../../../lib/prisma'
+
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+    const teamMemberMail = req.query.mail.toString()
+    
+    await prisma.team_members.delete({ 
```

Comment:
First, we have to make sure there is a team_member with this email, otherwise, we should return an error message.
Look at the 'api/prospect/create' to see an example of what I'm talking about.

Reply:
Why do I need to check the existence of the email address in both files? Isn't it enough to check it in one place?

Reply:
what is the other file that you're referring to?

Reply:
pages > team-members > profile > [mail]

Reply:
You're right, we can do everything there. Also, I think you can do it without useEffect just by using getServerSideProps